shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -113,7 +113,7 @@ EmbedderConfigBuilder::EmbedderConfigBuilder(
     SetSemanticsCallbackHooks();
     SetLogMessageCallbackHook();
     SetLocalizationCallbackHooks();
-    AddCommandLineArgument("--disable-observatory");
+    AddCommandLineArgument("--disable-vm-service");
```

Comment:
You should also support `--disable-observatory` until the tool is updated.

Reply:
Oh this is in `tests`, not sure if that's necessary then.

Reply:
Yeah, this is internal to the engine so we should be good to just change this as part of this PR.